library/core/src/main/java/com/google/android/exoplayer2/drm/HttpMediaDrmCallback.java

```diff
@@ -47,29 +47,36 @@
   }
 
   private final HttpDataSource.Factory dataSourceFactory;
-  private final String defaultUrl;
+  private final String defaultLicenseUrl;
+  private final Boolean forceDefaultLicenseUrl;
   private final Map<String, String> keyRequestProperties;
 
   /**
-   * @param defaultUrl The default license URL.
+   * @param defaultLicenseUrl The default license URL.
+   * @param forceDefaultLicenseUrl If true, default license URL will be used in the license
+   *    acquisition process even do there is a license URL defined in the stream manifest.
    * @param dataSourceFactory A factory from which to obtain {@link HttpDataSource} instances.
    */
-  public HttpMediaDrmCallback(String defaultUrl, HttpDataSource.Factory dataSourceFactory) {
-    this(defaultUrl, dataSourceFactory, null);
+  public HttpMediaDrmCallback(String defaultLicenseUrl, Boolean forceDefaultLicenseUrl,
```

Comment:
Please can we keep the original constructor too, so you don't have to add false to any existing call sites?